docs/report/report.tex

```diff
@@ -44,39 +44,69 @@ \section{Einleitung}
 Da es sich um ein Open Source Projekt mit entsprechender Lizenz handelt, muss auch eine aussagekräftige, öffentliche Dokumentation verfasst werden.
-Macio stellt die benötigte Hardware zur Verfügung und unterstützt bei technischen Fragen.
-
-\subsubsection{Minimal Requirements}
-\begin{enumerate}
-  \item NFC-Tags lesen, schreiben und entschlüsseln
-  \item Mit Spotify Connect verbinden und arbeiten
-  \item Responsive UI konzeptionieren und umsetzen
-  \item Aussagekräftige Dokumentation mit Benutzerhandbuch
-\end{enumerate}
-\subsubsection{Stretch Goals}
-\begin{enumerate}
-  \item Sound Wiedergabe auf der Box selbst
-  \item Unterstützung anderer Musikdienste / Plugin-Subsystem
-  \item 3D-Modellierung und Print einer passenden Box
-  \item Cloud-Anbindung der Box, Auslieferung des UI aus der Cloud
-\end{enumerate}
+Macio stellt die benötigte Hardware zur Verfügung und unterstützt bei technischen Fragen.\\
+
+\subsubsection{Anforderungen}
+Aus der Projektskizze und der Produktvision lassen sich konkreter folgende Mindestanforderungen und entsprechende Klassifikationen an das Produkt ableiten:
+\begin{center}
+  \begin{tabular}[h]{c|c}
+    Anforderung & Klassifikation \\
+    \hline
+    NFC-Tags lesen, schreiben und entschlüsseln & funktional  \\
+    Mit Spotify Connect verbinden und arbeiten & funktional \\
+    Responsive UI konzeptionieren und umsetzen & funktional  \\
+    Aussagekräftige Dokumentation mit Benutzerhandbuch & nicht funkt.  \\
+  \end{tabular}
+\end{center}
+
+Um diese Anforderungen weiter zu definieren hat das Team hierfür User Stories erstellt.
+Durch die User Stories wurde sichergestellt, dass beide Parteien (der Kunde und die Entwickler) die Anforderungen auf gleiche Weise interpretieren und verstehen.
+Dabei entstand auch ein Priorisierung der Anforderungen bemessen an dem Return of Investment und dem Nutzen für den Endnutzer.
+Im weiteren Gespräch mit den Kunden ergeben sich weitere ergänzende Anforderungen an das Produkt:
+\begin{center}
+  \begin{tabular}[h]{c|c}
+    Anforderung & Klassifikation \\
+    \hline
+    Sound Wiedergabe auf der Box selbst & funktional  \\
+    Unterstützung anderer Musikdienste & funktional \\
+    3D-Modellierung und Print einer passenden Box & nicht funkt.  \\
+    Maximaler Kostenpunkt von 30€ & nicht funkt.  \\
+    Cloud-Anbindung der Box & funktional \\
+    Auslieferung des UI aus der Cloud & funktional \\
+    \end{tabular}
+\end{center}
+
+Durch die agile Entwicklung des Projektes sind diese Anforderungen iterativ von Sprint zu Sprint gewachsen.
+Neue implizite Anforderungen sind aus den bestehenden Anforderungen entstanden.
+Beispielsweise um den Kostenpunkt von maximal 30€ zu realisieren entstand die Idee einer kleinen kostengünstigeren Version, welche unter 30€ liegt und nur die Grundfunktionen umsetzt, und einer größerer umfangreicheren, die weitere Funktionen beinhaltet wie das Abspielen der Musik über einen eigenen Lautsprecher.
```

Comment:
```suggestion 
Um den Kostenpunkt von maximal 30€ zu realisieren entstand beispielsweise die Idee, einer kleinen kostengünstigeren Version, bei der die Kosten weniger als 30€ betragen und nur die Grundfunktionen umsetzt, und einer größerer umfangreicheren, die weitere Funktionen beinhaltet wie das Abspielen der Musik über einen eigenen Lautsprecher.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/466/commits/bac512992bc6f3232f89c898a0d23eebfb63243b